app/src/main/java/org/oppia/app/topic_index/TopicIndexActivity.kt

```diff
@@ -0,0 +1,15 @@
+package org.oppia.app.topic_index;
+
+import android.os.Bundle;
+import androidx.appcompat.app.AppCompatActivity;
+import org.oppia.app.R
+
+/** The activity will display All the topic's summary. */
```

Comment:
@veena14cs there are very minute mistakes in this javadoc comment like grammar and text-capitalization. 
This is something which is not that important but because we are just starting up with PRs its good if we follow best practices.